linkerd/tls/src/server/mod.rs

```diff
@@ -373,3 +365,38 @@ impl fmt::Display for NoServerTls {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use io::AsyncWriteExt;
+
+    use super::*;
+    use std::str::FromStr;
+
+    #[tokio::test]
+    async fn detect_buffered() {
+        let _ = tracing_subscriber::fmt::try_init();
```

Comment:
nit: for tests, we should prefer
```suggestion
        let _ = tracing_subscriber::fmt().with_test_writer().try_init();
```
which will enable libtest to capture the test's output and only print it when the test fails (perhaps this should just be the default behavior in tracing...).

Reply:
I tried this and it didn't work. I didn't have time to dig into why...